spec/preserve_spec.rb

```diff
@@ -44,6 +44,15 @@
     expect(json_response[:status]).to eq(nil)
   end
 
+  it 'applies callback conditional options' do
+    ParametersController.preserve(:status, only: :index, if: -> { false })
```

Comment:
I think we can get rid of `only: :index` here to make it more obvious that we're only testing the `if` option.